helpme/templates/helpme/ticket_detail.html

```diff
@@ -51,7 +51,7 @@ <h3 class="mt-3">
     <div class="text-primary font-weight-bold dropdown mb-2" id="most_recent">
       <a role="button" class="dropdown-toggle"> Most recent </a>
     </div>
-    <div class="container comment-thread">
+    <div class="container" style="height:375px; overflow-y: scroll;">
```

Comment:
didn't know if you tried this:  https://getbootstrap.com/docs/4.3/utilities/overflow/